derive_client/clients/async_client.py

```diff
@@ -49,7 +63,137 @@ def __init__(
 
         self.message_queues = {}
         self.connecting = False
-        # we make sure to get the event loop
+
+    @functools.cached_property
+    def bridge(self) -> BridgeClient:
+        return BridgeClient(env=self.env, account=self.signer, wallet=self.wallet, logger=self.logger)
+
+    async def prepare_deposit_to_derive(
+        self,
+        amount: float,
+        currency: Currency,
+        chain_id: ChainID,
+    ) -> PreparedBridgeTx:
+        """
+        Prepare a deposit transaction to bridge tokens to Derive.
+
+        This creates a signed transaction ready for submission but does not execute it.
+        Review the returned PreparedBridgeTx before calling submit_bridge_tx().
```

Comment:
no way these docstring are there anymore in current work (branch state I'm working on). I've explicitly documented what is exposed on AsynClient and won't, unnecessarily, pollute my code with docstrings